src/test/regress/expected/multi_mx_router_planner.out

```diff
@@ -538,7 +529,7 @@ HINT:  Set citus.task_executor_type to "task-tracker".
 -- subqueries are not supported in SELECT clause
 SELECT a.title AS name, (SELECT a2.id FROM articles_single_shard_hash_mx a2 WHERE a.id = a2.id  LIMIT 1)
 						 AS special_price FROM articles_hash_mx a;
-ERROR:  could not run distributed query with subquery outside the FROM clause
+ERROR:  could not run distributed query with subquery outside the FROM and WHERE clauses
```

Comment:
But, all the error messages in `ErrorIfQueryNotSupported()` starts with `could not`. I don't think it is a good idea to only change this? We probably should do it later?